.gitignore

```diff
@@ -26,39 +10,16 @@ datatracker.sublime-workspace
 /.Python
 /.settings
 /.tmp
-/attic
-/bin
-/bootstrap/node_modules
 /data
 /docker/docker-compose.extend-custom.yml
-/env
```

Comment:
I'm not aware of anyone attempting native installs for development anymore, but if there is anyone, this will be annoying